ci/build.py

```diff
@@ -147,30 +176,65 @@ def script_name() -> str:
     docker_binary = get_docker_binary(args.nvidiadocker)
 
     if args.list:
-        platforms = get_platforms()
-        print(platforms)
+        list_platforms()
 
     elif args.platform:
         platform = args.platform
-        if args.build:
+        if args.build or args.build_only:
             build_docker(platform, docker_binary)
+        if args.build_only:
+            return 0
         tag = get_docker_tag(platform)
         if command:
             container_run(platform, docker_binary, command)
+        elif args.docker_run:
+            print(container_run(platform, docker_binary, [""], True))
         else:
             cmd = ["/work/mxnet/ci/docker/runtime_functions.sh", "build_{}".format(platform)]
             logging.info("No command specified, trying default build: %s", ' '.join(cmd))
             container_run(platform, docker_binary, cmd)
 
-    else:
+    elif args.all:
         platforms = get_platforms()
         logging.info("Building for all architectures: {}".format(platforms))
         logging.info("Artifacts will be produced in the build/ directory.")
         for platform in platforms:
             build_docker(platform, docker_binary)
+            if args.build_only:
+                continue
             cmd = ["/work/mxnet/ci/docker/runtime_functions.sh", "build_{}".format(platform)]
-            logging.info("No command specified, trying default build: %s", ' '.join(cmd))
+            shutil.rmtree(buildir(), ignore_errors=True)
             container_run(platform, docker_binary, cmd)
+            plat_buildir = os.path.join(get_mxnet_root(), "build_{}".format(platform))
```

Comment:
Maybe "build.{}".format(platform)?

Reply:
it's a bash function with an underscore, not sure if you can put dots in a bash function

Reply:
I like the filename format with underscores